good

Thomas88s opened the pull request:
# Description
Added dropdown for attractions included in attractionsSelect.



Fixes # (issue)

## Type of change

Please delete options that are not relevant.


- [ ] New feature (non-breaking change which adds functionality)

# Testing Instructions


git checkout tw-dropdown2



# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

Thomas88s committed:
good

nicoletatum reviewed (dismissed):
looks great

Thomas88s committed:
corrections for attractions dropdown

nicoletatum reviewed (dismissed):
👍

dylanmorris51 commented on scripts/attractions/AttractionsSelect.js:
```diff
@@ -0,0 +1,43 @@
+            }
+        })
+        // console.log(customEvent)
+        eventHub.dispatchEvent(customEvent)
```
can you add ```console.log("Attractions Selection Successfully Dispatched!")```

morriscodez reviewed (dismissed):
Nice!

emilycentko reviewed (dismissed):
great!

morriscodez committed:
Merge branch 'main' into tw-dropdown2

morriscodez committed:
Update AttractionsSelect.js

added ```console.log("Attractions Selection Successfully Dispatched!")``` to line 13 so we can visibly see in the console that the click event is dispatching as expected when clicked

morriscodez reviewed (approved):
Nice!
